LINK/etc/default/evm

```diff
@@ -2,23 +2,15 @@
 # Description: Sets the environment for scripts and console users
 #
 
-export RAILS_ENV=production
-export APPLIANCE=true
-export HOME=${HOME:-/root}
-# Force ExecJS to use node
-export EXECJS_RUNTIME='Node'
-# workaround for virtual memory spike observed with RHEL6
-export MALLOC_ARENA_MAX=1
-# Location of certificates and provider keys
-export KEY_ROOT=/var/www/miq/vmdb/certs
-
-[[ -s /etc/default/evm_ruby ]] && source /etc/default/evm_ruby
-[[ -s /etc/default/evm_postgres ]] && source /etc/default/evm_postgres
-source /etc/default/evm_production
+for prop in /etc/default/evm.properties /etc/default/evm_production ; do
+  if [[ -f ${prop} ]] ; then 
+    # lines starting with a semi or a # are ignored
```

Comment:
why semicolon?  is that some standard.  also for readability

```suggestion
    # lines starting with a `;` or a `#` are ignored
```

Reply:
This comes from the format of an EnvironmentFile: https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=

> Empty lines, lines without an "=" separator, or lines starting with ; or # will be ignored, which may be used for commenting.

Reply:
I'm surprised there isn't a tool that does this already...